python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -244,13 +244,18 @@ def train_epochs(self, data_creator, epochs=1, batch_size=32, profile=False,
         self.val_loader = val_loader
         self.call_hook(callbacks=callbacks, fn_name="before_run")
 
+        if max_steps is not None:
+            epochs = math.ceil(max_steps / len(self.train_loader))
```

Comment:
So `max_step` indicates a global step?

Reply:
global step is runner status, max steps is a training option for just training specific steps.

Reply:
Yes, I mean the max_step is **compared** with the global step across all ranks?